python/core/more-on-lists/using-a-list-as-a-stack.md

```diff
@@ -31,7 +31,7 @@ parent: built-in-list-methods
 
 The `stack` abstract data structure is widely used in computer science. This collection is implemented to best fit the **LIFO** principle ( **last in**, **first out** ). Sometimes we need one data structure that behaves in such a manner.
```

Comment:
I don't think any of these actually need code blocks around them because they're just English words, not words that are part of code.

instead of changing `list`'s to `list`s, why not just use lists? Same goes for `python` -> python and `stack` -> stack.